corehq/apps/commtrack/views.py

```diff
@@ -293,12 +296,35 @@ class RebuildStockStateView(BaseCommTrackManageView):
 
     @property
     def page_context(self, **kwargs):
```

Comment:
should we consider limiting what this returns explicitly? just worried about someone trying to load this page on some big domain and then the server running out of memory.